docs/components/best-practices/architecture/deciding-about-your-stack.md

```diff
@@ -9,83 +9,77 @@ tags:
+   1. **Sign up:** Users need to sign up for Camunda 8 SaaS. This can be done through the [Camunda SaaS registration page.](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral)
+   2. **Create a cluster:** After signing up, users should create a cluster. They can follow the guide on [Creating a Cluster in Camunda 8](/guides/create-cluster.md) which provides step-by-step instructions on setting up a new cluster in the Camunda 8 environment.
 
-1. Use [Maven](https://maven.apache.org/) as a build tool.
+2. **Spring Boot:**
 
-1. Use your favorite IDE, for example Visual Studio Code, IntelliJ or Eclipse.
+   1. **Develop a Spring Boot application:** Users should develop their process solutions as [Spring Boot](https://spring.io/projects/spring-boot) applications. This involves setting up a new Spring Boot project, either manually or using tools like [Spring Initializr](https://start.spring.io/).
+   2. **Integrate Spring Zeebe:** Integrate [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) into the Spring Boot project by adding necessary dependencies to the project’s pom.xml file and configuring the application to use Camunda services.
 
-1. Use [Open JDK 17](https://jdk.java.net/17/) as Java runtime.
+3. **Maven:**
 
-1. Model the processes with the [Camunda Modeler](https://camunda.org/download/modeler/).
+   1. **Use Maven for build management:** [Maven](https://maven.apache.org/) should be used to manage the build lifecycle of the application.
 
-1. Add your process models and all Java code to the project.
+4. **IDE Selection:**
 
-To run the process application _in production_:
+   1. **Choose an IDE:** Select an Integrated Development Environment (IDE) that supports Java development, Maven, and Spring Boot. Frequently used options include Visual Studio Code, IntelliJ IDEA, or Eclipse.
 
-3. Run the process application by copying the `jar` file to the server and start it with `java -jar YourProcessApplication.jar`. This is most often done via Docker.
+5. **Java runtime:**
 
-Refer to our [example application](https://github.com/camunda-community-hub/camunda-cloud-examples/tree/main/twitter-review-java-springboot).
+   1. **Install OpenJDK 17:** Install and use OpenJDK 17 as your Java runtime environment. Download it from the [official JDK 17 download page](https://jdk.java.net/17/).
 
-We decided on this stack for the following reasons:
+6. **Modeling:**
 
-- All components are open source and easily available.
-- SaaS is the easiest way to consume capabilities like a workflow engine.
-- Spring Boot is currently the most adopted way of building Java applications.
-- Spring Boot applications are easy to customize as well as easy to roll out into test and production environments, either on-premise or in the cloud.
+   1. **Use Camunda desktop Modeler:** Download and use the Camunda Modeler for designing and modeling business processes. The Modeler is available here.
```

Comment:
The rest of this page is SaaS-first - why not Web Modeler?

Reply:
This is deliberate, the greenfield stack will expand to cover self managed, also customers looking to implement using a spring boot application currently also align better with the desktop modeler for now. We wanted to avoid providing multiple choices at this stage but plan on adjusting as the document grows

Here link is missing, ill update that